### DISCUSSION TITLE
CVAT still displays version 1.6 after update to 2.1 - and no packages are rebuild

### DISCUSSION THREAD

**Discussion by thomasm2film** [cvat-ai/cvat#4467] [General]
Just tried updating my 1.6 server to 2.1 using

git clone https://github.com/opencv/cvat

When I do company@cvat:~/cvat$ docker-compose build

I get

WARNING: The no_proxy variable is not set. Defaulting to a blank string.
cvat_redis uses an image, skipping
cvat_opa uses an image, skipping
cvat_db uses an image, skipping
traefik uses an image, skipping
cvat uses an image, skipping
cvat_ui uses an image, skipping

And version is still 1.6 in web UI after docker-compose up -d

When I cat CHANGELOG.MD, it shows 2.1...

What could be wrong here?

Thanks

**Comment by haimat** [comment]
We have exactly the same issue - any new insights on this?

**Reply by thomasm2film** [reply]
I tried docker-compose pull - that downloaded images and now took me to version 2.0. Perhaps because 2.1 is unreleased?

**Reply by haimat** [reply]
Thanks. Did you also run `build` after `pull`?
If not, how does the DB have been upgraded to the new version?

**Reply by thomasm2film** [reply]
I tried build after pull - and that gave me this again... But seems to be working now...

cvat_redis uses an image, skipping
cvat_opa uses an image, skipping
cvat_db uses an image, skipping
traefik uses an image, skipping
cvat uses an image, skipping
cvat_ui uses an image, skipping

**Reply by thomasm2film** [reply]
Basically this is my "update script"

```
cd cvat
docker-compose down
cd ..
mv cvat cvat_old_$(date +"%d%m%y")
git clone https://github.com/opencv/cvat
cp _cvat_backup_files/* cvat/
cd cvat
export CVAT_HOST=cvat.domain.com
docker-compose pull
docker-compose -f docker-compose.yml -f docker-compose.https.yml -f docker-compose.override.yml up -d
```

**Reply by thomasm2film** [reply]
I use custom SSL certs - that's why the extra files

**Reply by haimat** [reply]
Thanks, that did the trick for me as well 👍

**Comment by azhavoro** [comment]
@thomasm2film Hello,
1. as you can see in the build log, you are not building images. This happens  because the default compose configuration does not contain a build section and you do not include the `docker-compose.dev.yml` configuration with `-f` option.
2. `docker-compose up -d` creates containers from images that exist on your server (or pulls if no images is present), so if you want to use the latest published version (v2.0.0), update your local images with `docker-compose pull`

**Reply by thomasm2film** [reply]
Ahh great thanks @azhavoro

**Reply by thomasm2film** [reply]
Might be included in https://openvinotoolkit.github.io/cvat/docs/faq/ for reference.

**Reply by Bradley01429** [reply]
@thomasm2film Pull Request: https://github.com/openvinotoolkit/cvat/pull/4562. Has been created to document the required step when updating CVAT. Hopefully, this will make it easier for people to update CVAT in the future :)

**Comment by nmanovic** [comment]
@thomasm2film , could you please help us and contribute to FAQ?

**Comment by haimat** [comment]
Hey guys, when will this PR be accepted in order to update CVAT documentation?
